suse_migration_services/units/prepare.py

```diff
@@ -101,6 +101,24 @@ def main():
     cloud_register_metadata = os.sep.join(
         [root_path, 'var', 'lib', 'cloudregister']
     )
+    zypper_log_path = os.sep.join(
+        [root_path, 'var', 'log', 'zypper.log']
+    )
+    if os.path.exists(zypper_log_path):
+        zypper_host_log_file = zypper_log_path.replace(root_path, '')
+        if not os.path.exists(zypper_host_log_file):
+            with open(zypper_host_log_file, 'w'):
+                # we bind mount the system zypper log file but the mount target
+                # does not exist. Create it as empty file prior bind mounting
+                pass
+    try:
+        Command.run(
+            ['mount', '--bind', zypper_log_path, '/var/log/zypper.log']
```

Comment:
avoid duplication of code: '/var/log/zypper.log' is zypper_host_log_file

Reply:
Fixed